src/cloudsc_fortran/dwarf_cloudsc.F90

```diff
@@ -19,6 +19,8 @@ PROGRAM DWARF_CLOUDSC
 USE YOMCST   , ONLY : YRCST 
 USE YOETHF   , ONLY : YRTHF
 
+USE OMP_LIB
+
```

Comment:
Also, guard with `#ifdef _OPENMP`, and in l.53 below

Reply:
resolved in next commit